packages/core/src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -254,6 +256,27 @@ export default class BraintreeHostedForm {
         }
     }
 
+    private _mapErrors(fields: BraintreeHostedFieldsState['fields']): BraintreeFormErrorsData {
+        const errors: BraintreeFormErrorsData = {};
```

Comment:
Do we need to assign the type here?

Reply:
I don't think so, what is the drawback to assign type here?